com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/MockCloudFoundryClient.java

```diff
@@ -751,4 +751,19 @@ public void associateManagerWithSpace(String orgName, String spaceName, String s
         return null;
     }
 
+    @Override

```

Comment:
Why not delete the MockCloudFoundryClient.java? It is not used anymore as I see.

Reply:
Ok i will have that in mind and try to remove in new commit.